gulpfile.js

```diff
@@ -177,6 +183,20 @@ pipes.buildArtifacts = function () {
 
 // == TASKS ========
 
+gulp.task('bump', function() {
+    var pkg = getPackageJson();
+
+    var newVer = pkg.version;
+
+    gulp.src('./bower.json')
+    .pipe(bump({version: newVer}))
+    .pipe(gulp.dest('./'));
+
+    gulp.src('./app/modules/version.json')
+    .pipe(bump({version: newVer + '-' + process.env.BUILD_NUMBER}))
```

Comment:
'process' isn't defined.  I tried to run this and I just kept getting "version 0.1.0-undefined" in the About modal.